bin/caql

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import click
+import math
+import json
+import sys
+from datetime import datetime
+from pathlib import Path
+
+from circonusapi import circonusdata
+
+@click.group()
+@click.option("-e", '--endpoint', default=None)
+@click.option("-a", '--account', default=None)
+@click.option("-t", '--token', default=None)
+@click.option("-c", '--config', default=None)
+@click.pass_context
+def cli(ctx, endpoint, account, token, config):
+    if config:
+        with open(Path("~/.caqlrc").expanduser(), "r") as fh:
```

Comment:
I don't see where `~/.caqlrc` is being written to?

Reply:
This file has to be created manually. Will document this once it's a little matured.